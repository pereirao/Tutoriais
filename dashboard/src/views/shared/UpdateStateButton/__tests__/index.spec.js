import React from 'react';

import order_factory from '../../../../business/order/__tests__/order.factory';
// NOTE: the named export isn't a connected component
import { UpdateStateButton } from '../index';

it('renders inline correctly', () => {
  expect(shallow(<UpdateStateButton inline order={order_factory.build()} />)).toMatchSnapshot();
});
it('renders non-inline correctly', () => {
  expect(shallow(<UpdateStateButton inline={false} order={order_factory.build()} />)).toMatchSnapshot();
});
