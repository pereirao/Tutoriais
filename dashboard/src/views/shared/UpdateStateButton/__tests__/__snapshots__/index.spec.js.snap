// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders inline correctly 1`] = `
<div
  className="updateStateButton inlineButton"
  onClick={[Function]}
>
  <p
    className="inlineMarkAs"
  >
    MARK AS
  </p>
  <p
    className="inlineNextState"
  >
    CONFIRM ORDER
  </p>
</div>
`;

exports[`renders non-inline correctly 1`] = `
<div
  className="updateStateButton"
  onClick={[Function]}
>
  <p
    className="mainText"
  >
    CONFIRM ORDER
  </p>
</div>
`;
